.github/dependabot.yml

```diff
@@ -0,0 +1,7 @@
+version: 2
+enable-beta-ecosystems: true
+updates:
+- package-ecosystem: docker
```

Comment:
Do we also plan to scan go.mod?

Reply:
Dependabot can indeed generate alerts for Go projects without an explicit dependabot.yml file. It does this by automatically scanning repositories for supported manifest files, including go.mod.

We are already getting alerts for go.mod file.

Reply:
I got to know that writing yml file will disable the default alerts, that's why adding more alerts.